pkg/platform/packet/packet.go

```diff
@@ -237,6 +252,25 @@ func (c *config) terraformSmartApply(ex *terraform.Executor, dnsProvider dns.DNS
 	return ex.Apply()
 }
 
+// terraformCreateReservations creates nodes that use a specific hardware
+// reservation ID
+func (c *config) terraformCreateReservations(ex *terraform.Executor) error {
+	arguments := []string{"apply", "-auto-approve"}
+
+	// Create workers that use specific UUIDS as hardware reservation
+	for _, w := range c.WorkerPools {
+		if len(w.ReservationIDs) > 0 {
+			arguments = append(arguments, fmt.Sprintf("-target=module.worker-%v.packet_device.nodes", w.Name))
+		}
+	}
+
+	// Create controllers that use specific UUIDS as hardware reservation
+	if len(c.ReservationIDs) > 0 {
+		arguments = append(arguments, fmt.Sprintf("-target=module.packet-%v.packet_device.controllers", c.ClusterName))
+	}
+
+	return ex.Execute(arguments...)
+}
```

Comment:
Functions should be separated with blank line.

Reply:
Fixed